src/Fixer/Whitespace/NoTrailingWhitespaceFixer.php

```diff
@@ -70,15 +70,6 @@ public function getDefinition()
     /**
      * {@inheritdoc}
      */
-    public function getPriority()
```

Comment:
the comment is specific for the fixer extending the abstract, so that would not be possible (or the same)